tldr/cross_file_calls.py

```
@@ -2409,6 +2411,124 @@ def _get_php_class_context(node, source: bytes) -> str | None:
+                        for cc in arg_child.children:
+                            if cc.type == "identifier":
+                                return source[cc.start_byte:cc.end_byte].decode("utf-8")
+                    elif arg_child.type == "identifier":
+                        # def func_name do (no args)
+                        return source[arg_child.start_byte:arg_child.end_byte].decode("utf-8")
+                    elif arg_child.type == "binary_operator":
+                        # def func_name \\ default_value
+                        for cc in arg_child.children:
+                            if cc.type == "identifier":
+                                return source[cc.start_byte:cc.end_byte].decode("utf-8")
+                            elif cc.type == "call":
```

Ruff (0.15.5)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)